lib/AST/Availability.cpp

```diff
@@ -330,6 +330,10 @@ AvailabilityContext ASTContext::getConcurrencyAvailability() {
   return getSwift55Availability();
 }
 
+AvailabilityContext ASTContext::getDistributedAvailability() {
+  return getSwift55Availability(); // FIXME(distributed): getSwiftFutureAvailability perhaps? The module is not shipping in any release so far so it does not matter for real though.
+}
```

Comment:
So for now we're just riding on 5.5 since the module does not ship anyway (it is removed from xcode distributions) and the types are only touchable when the experimental flag is passed. @DougGregor was of the opinion this is fine for now.

Though I notice the getSwiftFutureAvailability just now and perhaps that would be nicer here...?

Usage would mean we have to do 99 everywhere for ALL our types on linux even as well though right...? We kind of have to anyway... Maybe another PR to change to this Future is worth it or not?